shell/shared/admin.js

```diff
@@ -409,7 +409,16 @@ if (Meteor.isClient) {
     },
     userIsGuest: function () {
       return !this.isAdmin && !this.signupKey;
-    }
+    },
+    userQuota: function () {
+      return (typeof this.quota === "number") ? prettySize(this.quota) : "";
```

Comment:
Consider returning "unlimited" or something equivalent instead of displaying an empty string.